### DISCUSSION TITLE
EDIT: This just doesnt work...?

### DISCUSSION THREAD

**Discussion by Micheal755** [UltimaHoarder/UltimaScraper#1960] [Q&A]
I got geckodriver, cookies and login all that. The app opens up on my firefox browser and I log in. But I still cant see anyone's content on OnlyFans it looks normal. 
![image](https://user-images.githubusercontent.com/121937646/210824504-7c149077-336c-44ec-93ac-64ef013054c3.png)
![image](https://user-images.githubusercontent.com/121937646/210824725-8556644e-f975-4d98-ae8b-b547e43d151a.png)
??? P.S. I added my email and password in the files with the cookies so idk what's wrong

**Comment by Vahodan** [comment]
I actually have the same issue I placed the Auth id and the user agent exactly as described in the instructions, it kept saying access denied on all attempts, I also tried to download geckodriver and place it under the extras folder. Nothing worked, I think there might be a problem with the webpage and the authentication process because I also tried to fill the email and password but it did not work either.

Hope someone can shine some light on if this code is still working because I saw there was an update on 7.6 but now it is 7.4 so maybe it is outdated.

**Comment by elephanticefire** [comment]
Shot in the dark here, are you actually subbed to the profile you're attempting to scrape? If not, there is your access error.
